src/device/pf_pdport.c

```diff
@@ -51,12 +52,17 @@ static int pf_pdport_load (pnet_t * net)
          "PDPORT(%d): Did read PDPort settings from nvm.\n",
          __LINE__);
 
-      memcpy (&net->port[0].check.peer, &file_port, sizeof (pnet_pdport_nvm_t));
+      p_port_data = pf_port_get_port_data (net, PNET_PORT_1);
+
+      memcpy (
+         &p_port_data->pdport.check.peer,
+         &file_port,
+         sizeof (pnet_pdport_nvm_t));
```

Comment:
```suggestion
      p_port_data->pdport.check.peer = file_port.peer;
```
Note that p_port_data->pdport.check.peer is not the same type as pnet_pdport_nvm_t.

Reply:
Done